RoddyCore/test/de/dkfz/roddy/plugins/LibrariesFactoryTest.groovy

```diff
@@ -139,17 +146,80 @@ public class LibrariesFactoryTest {
         assert mapOfAvailablePlugins["B"]["1.0.3-0"]?.previousInChain == mapOfAvailablePlugins["B"]["1.0.2-2"] && mapOfAvailablePlugins["B"]["1.0.3-0"]?.previousInChainConnectionType == PluginInfo.PluginInfoConnection.EXTENSION
     }
 
-    public static PluginInfoMap callLoadMapOfAvailablePlugins(List<File> additionalPluginDirectories = []) {
-        List<File> pluginDirectories = [new File(Roddy.getApplicationDirectory(), "/dist/plugins/")]
-        pluginDirectories += additionalPluginDirectories
+    @BeforeClass
+    public static void setupTestDataForPluginLoaderTest() {
```

Comment:
How about creating example files like for the workflow exec test? This way we would only need to copy the folders / files and modify e.g. readability and so.